client/my-sites/checkout/checkout-thank-you/marketplace/style.scss

```diff
@@ -87,16 +85,21 @@
 				line-height: 20px;
 				font-size: $font-body-small;
 				color: var(--studio-gray-60);
+				margin-bottom: 16px;
```

Comment:
This is needed to override the default styling in https://github.com/Automattic/wp-calypso/blob/trunk/client/components/thank-you/style.scss#L50